test/origin_simulator/multi_route_origin_simulator_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule MultiRouteOriginSimulatorTest do
+  use ExUnit.Case
+  use Plug.Test
+
+  setup do
+    OriginSimulator.Simulation.restart()
+    Process.sleep(10)
+  end
+
+  describe "GET pages" do
```

Comment:
I'm not as familiar with Elixir conventions but often I've seen the describe blocks detailing the class/module. So in this case something like

```
describe "MultiRouteOriginSimulator" do
    test "GET pages for a wildcard route should return 200 OK" do
```

Reply:
Thanks @astalker, I made the description changes and reorganisation for the tests according to our chat, adopted a little bit of the cucumber format. Committed the change in https://github.com/bbc/origin_simulator/pull/22